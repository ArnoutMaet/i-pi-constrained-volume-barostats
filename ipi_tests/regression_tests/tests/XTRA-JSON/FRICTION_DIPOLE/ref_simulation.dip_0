 #*EXTRAS*# Step:           0  Bead:      0  
   -13.50071746           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      0  
   -14.35911642           0.00000000           0.00000000
 #*EXTRAS*# Step:           2  Bead:      0  
   -15.24351157          -0.00000000           0.00000000
 #*EXTRAS*# Step:           3  Bead:      0  
   -16.01962737          -0.00000000           0.00000000
 #*EXTRAS*# Step:           4  Bead:      0  
   -16.66858746           0.00000000           0.00000000
 #*EXTRAS*# Step:           5  Bead:      0  
   -17.22188943           0.00000000           0.00000000
 #*EXTRAS*# Step:           6  Bead:      0  
   -17.66323303           0.00000000           0.00000000
 #*EXTRAS*# Step:           7  Bead:      0  
   -18.02558788           0.00000000           0.00000000
