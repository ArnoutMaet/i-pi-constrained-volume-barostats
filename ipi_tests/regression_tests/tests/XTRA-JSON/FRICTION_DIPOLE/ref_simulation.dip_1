 #*EXTRAS*# Step:           0  Bead:      1  
    -1.68910806           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      1  
    -2.03928208           0.00000000           0.00000000
 #*EXTRAS*# Step:           2  Bead:      1  
    -2.48992471           0.00000000           0.00000000
 #*EXTRAS*# Step:           3  Bead:      1  
    -2.98245102           0.00000000           0.00000000
 #*EXTRAS*# Step:           4  Bead:      1  
    -3.46903058           0.00000000           0.00000000
 #*EXTRAS*# Step:           5  Bead:      1  
    -3.93650390           0.00000000           0.00000000
 #*EXTRAS*# Step:           6  Bead:      1  
    -4.34012681           0.00000000           0.00000000
 #*EXTRAS*# Step:           7  Bead:      1  
    -4.68821433           0.00000000           0.00000000
