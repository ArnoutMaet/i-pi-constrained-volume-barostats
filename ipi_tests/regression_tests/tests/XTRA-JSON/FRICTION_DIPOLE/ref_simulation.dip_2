 #*EXTRAS*# Step:           0  Bead:      2  
    -0.10878315           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      2  
    -0.20142075           0.00000000           0.00000000
 #*EXTRAS*# Step:           2  Bead:      2  
    -0.38972612           0.00000000           0.00000000
 #*EXTRAS*# Step:           3  Bead:      2  
    -0.67597172          -0.00000000           0.00000000
 #*EXTRAS*# Step:           4  Bead:      2  
    -1.03572261           0.00000000           0.00000000
 #*EXTRAS*# Step:           5  Bead:      2  
    -1.45001843           0.00000000           0.00000000
 #*EXTRAS*# Step:           6  Bead:      2  
    -1.86118335           0.00000000           0.00000000
 #*EXTRAS*# Step:           7  Bead:      2  
    -2.25538380           0.00000000           0.00000000
