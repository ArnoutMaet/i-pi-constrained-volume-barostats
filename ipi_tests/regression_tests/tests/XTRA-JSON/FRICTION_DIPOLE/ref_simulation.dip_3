 #*EXTRAS*# Step:           0  Bead:      3  
     0.00000000           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      3  
    -0.00350033           0.00000000          -0.00000000
 #*EXTRAS*# Step:           2  Bead:      3  
    -0.04347490          -0.00000000          -0.00000000
 #*EXTRAS*# Step:           3  Bead:      3  
    -0.16423104           0.00000000           0.00000000
 #*EXTRAS*# Step:           4  Bead:      3  
    -0.38898755          -0.00000000           0.00000000
 #*EXTRAS*# Step:           5  Bead:      3  
    -0.72746158          -0.00000000           0.00000000
 #*EXTRAS*# Step:           6  Bead:      3  
    -1.13747897          -0.00000000           0.00000000
 #*EXTRAS*# Step:           7  Bead:      3  
    -1.59467084          -0.00000000           0.00000000
