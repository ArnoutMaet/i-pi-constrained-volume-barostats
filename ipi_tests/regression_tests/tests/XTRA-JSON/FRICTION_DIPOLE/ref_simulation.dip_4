 #*EXTRAS*# Step:           0  Bead:      4  
     0.10878315           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      4  
     0.05891994          -0.00000000          -0.00000000
 #*EXTRAS*# Step:           2  Bead:      4  
     0.04769195           0.00000000           0.00000000
 #*EXTRAS*# Step:           3  Bead:      4  
     0.04501924           0.00000000           0.00000000
 #*EXTRAS*# Step:           4  Bead:      4  
     0.00531112           0.00000000           0.00000000
 #*EXTRAS*# Step:           5  Bead:      4  
    -0.11941570           0.00000000           0.00000000
 #*EXTRAS*# Step:           6  Bead:      4  
    -0.34996787          -0.00000000           0.00000000
 #*EXTRAS*# Step:           7  Bead:      4  
    -0.69277547          -0.00000000           0.00000000
