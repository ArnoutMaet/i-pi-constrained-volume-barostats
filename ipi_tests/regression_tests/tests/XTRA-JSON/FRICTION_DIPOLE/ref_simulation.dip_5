 #*EXTRAS*# Step:           0  Bead:      5  
     1.68910806           0.00000000           0.00000000
 #*EXTRAS*# Step:           1  Bead:      5  
     1.39654968           0.00000000           0.00000000
 #*EXTRAS*# Step:           2  Bead:      5  
     1.14646690          -0.00000000           0.00000000
 #*EXTRAS*# Step:           3  Bead:      5  
     0.96838013          -0.00000000           0.00000000
 #*EXTRAS*# Step:           4  Bead:      5  
     0.87251971          -0.00000000           0.00000000
 #*EXTRAS*# Step:           5  Bead:      5  
     0.84175099          -0.00000000           0.00000000
 #*EXTRAS*# Step:           6  Bead:      5  
     0.84189214          -0.00000000           0.00000000
 #*EXTRAS*# Step:           7  Bead:      5  
     0.82542303          -0.00000000           0.00000000
