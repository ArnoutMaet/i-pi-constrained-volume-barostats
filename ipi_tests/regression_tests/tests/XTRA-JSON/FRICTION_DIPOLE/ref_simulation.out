# column   1     --> step : The current simulation time step.
# column   2     --> potential{electronvolt} : The physical system potential energy.
    0.00000000e+00    -1.07641831e-01   
    1.00000000e+00    -1.08327310e-01   
    2.00000000e+00    -1.11007966e-01   
    3.00000000e+00    -1.15230571e-01   
    4.00000000e+00    -1.20494968e-01   
    5.00000000e+00    -1.26583077e-01   
    6.00000000e+00    -1.32841447e-01   
    7.00000000e+00    -1.39387929e-01   
